archunit-tooling-test/src/main/java/com/tngtech/archunit/tooling/engines/gradle/GradleProjectLayout.java

```diff
@@ -0,0 +1,49 @@
+package com.tngtech.archunit.tooling.engines.gradle;
+
+import java.nio.file.Path;
+
+import com.tngtech.archunit.tooling.utils.JavaProjectLayout;
+
+public class GradleProjectLayout extends JavaProjectLayout<GradleProjectLayout.GradleProject> {
+
+    public GradleProjectLayout(
+            String basePackage,
+            String buildManifestLocation,
+            String compiledClassesLocation,
+            String sourcesLocation,
+            String buildToolResourcesLocation) {
+        super(basePackage, buildManifestLocation, compiledClassesLocation, sourcesLocation, buildToolResourcesLocation);
+    }
+
+    @Override
+    protected String getCompiledTestClassesTargetDirectory() {
+        return "build/classes/java/test";
+    }
+
+    @Override
+    protected String getBuildToolWrapperDirectory() {
+        return "gradle";
```

Comment:
The Gradle wrapper is not really used for now. 

I guess it can be used once the Gradle PR is merged and the adjusted version is available from Gradle itself